pkgs/applications/plasma-mobile/qmlkonsole.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, mkDerivation
+
+, cmake
+, extra-cmake-modules
+
+, kconfig
+, ki18n
+, kirigami-addons
+, kirigami2
+, kcoreaddons
+, qtquickcontrols2
+, kwindowsystem
+, qmltermwidget
+}:
+
+mkDerivation {
+  pname = "qmlkonsole";
+
+  nativeBuildInputs = [
+    cmake
+    extra-cmake-modules
+  ];
+
+  buildInputs = [
+    kconfig
+    ki18n
+    kirigami-addons
+    kirigami2
+    qtquickcontrols2
+    kcoreaddons
+    kwindowsystem
+    qmltermwidget
+  ];
+
+  meta = with lib; {
+    description = "Terminal app for Plasma Mobile";
+    homepage = "https://invent.kde.org/plasma-mobile/qmlkonsole";
+    license = with licenses; [ gpl2Plus cc0 ];
```

Comment:
Considering some source files specifically state GPL 3.0 or later, too, it will have to be added to the list

 - https://invent.kde.org/plasma-mobile/qmlkonsole/-/blob/761abcc5e0aa26dbc4617060859d49a603da882b/src/abouttype.h
 - https://invent.kde.org/plasma-mobile/qmlkonsole/-/blob/761abcc5e0aa26dbc4617060859d49a603da882b/src/abouttype.cpp